rclpy/test/test_switch_rmw_implementation.py

```diff
@@ -0,0 +1,101 @@
+# Copyright 2016 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import importlib
+from multiprocessing import Array
+from multiprocessing import Process
+import os
+
+
+def f_rmw(result):
+    import rclpy
+    import rclpy.selector
```

Comment:
oh, I'm guessing this test is leftover after @wjwwood's recent commit because it still uses `selector`, which William removed.